tests/notebook-demo.ipynb

```diff
@@ -1,183 +1,312 @@
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "NzERPjTHzaHI",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "NzERPjTHzaHI",
+        "outputId": "71a09d8e-d429-4bcc-d751-c40c356d7ed8"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+            "100  226k  100  226k    0     0  85880      0  0:00:02  0:00:02 --:--:-- 85902\n"
+          ]
+        }
+      ],
+      "source": [
+        "%%bash\n",
+        "# Download the Demo 3 model\n",
+        "curl 'https://drive.usercontent.google.com/download?id=10zeI0xDk4g_I1e5xFwiC7e74Z21uYgWN&export=download&authuser=0&confirm=t' -o DemoS_003.zip\n",
+        "unzip DemoS_003.zip\n",
+        "# Clone the TIMES source code\n",
+        "git clone --filter=blob:none https://github.com/etsap-TIMES/TIMES_model TIMES_model\n",
+        "pushd TIMES_model/\n",
+        "git checkout b488fb07f0899ee8b7e710c230b1a9414fa06f7d\n",
+        "popd"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "199657e6",
+      "metadata": {},
+      "source": [
+        "## Run a model scenario"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "id": "65179b0a",
+      "metadata": {
+        "id": "65179b0a"
+      },
+      "outputs": [],
+      "source": [
+        "from os import path\n",
+        "\n",
+        "import gdx2py\n",
+        "from gamspy_base import directory\n",
+        "\n",
+        "import xl2times.main as xt\n",
+        "from xl2times.utils import run_gams, setup_logger"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 25,
+      "id": "f4e18642",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "f4e18642",
+        "outputId": "dcd21280-d7f5-424f-a016-6a8587ff7dce"
+      },
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "/usr/local/lib/python3.11/dist-packages/xl2times/transforms.py:1276: FutureWarning: Setting an item of incompatible dtype is deprecated and will raise an error in a future version of pandas. Value 'EOH' has dtype incompatible with float64, please explicitly cast to a compatible dtype first.\n",
+            "  df.loc[\n",
+            "Applying transformations from ~TFM_UPD in SYSSETTINGS: 100%|██████████| 2/2 [00:00<00:00, 97.03it/s]\n"
+          ]
+        }
+      ],
+      "source": [
+        "setup_logger(level=0)\n",
+        "input_dir = \"/content/DemoS_003/\"\n",
+        "output_dir = \"/content/DemoS_003-all-out/\"\n",
+        "times_dir = \"/content/TIMES_model/\"\n",
```

Comment:
These paths only work on Colab. Wonder if there's a way to do it so that it can also work locally, but a quick google says that it's hard in general to figure out whether a notebook is running on Colab / locally..